### DISCUSSION TITLE
Multiple Selection in Tree Component?

### DISCUSSION THREAD

**Discussion by Ri-gers** [DominoKit/domino-ui#980] [Q&A]
What are your thoughts on having a Tree variant where multiple items could be selected instead of a single 'active' one? For example you could check a checkbox next to a root node and thus all children would be selected. Parent nodes could have a state like selected/not selected/indeterminate (somewhat similar to the Trees here: https://dominokit.com/solutions/domino-ui/v2/docs/data-table/tree-eagerhttps://dominokit.com/solutions/domino-ui/v2/docs/data-table/tree-eager). 
A usecase could be that you configure permissions for users - if you check a root node, a user is allowed to do everything, if only a child is checked in the Tree, they can only perform a subaction etc.
Thoughts on this?

**Comment by FrankHossfeld** [comment]
Sounds great!

We had a similar request. It ends up with the TreePlugin for the DataTable domino-ui is offering for all of us now. We paid DominoKit for the implementation and agreed to open source it. I think, it is a fair trade to get things down.

**Reply by Ri-gers** [reply]
Oh that sounds great, thanks for the info and supporting this! :)

**Comment by vegegoku** [comment]
Can you please file this as an issue? 
I prefer if these are directly filed as issues instead of discussions. thanks
